### PR TITLE
[HackaDoc] new article for messaging overview

### PR DESCRIPTION
**Pull request by djrosanova**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@djrosanova__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (darosa). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: articles/service-bus-messaging-overview.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	ms.service="service-bus"
+	ms.workload="tbd"
+	ms.tgt_pltfrm="na"
+	ms.devlang="multiple"
+	ms.topic="article"
+	ms.date="02/20/2015"
+	ms.author="sethm"/>
+
+
+# Service Bus Messaging: Flexible Data Delivery in the Cloud
```
Comment (mollybostic) [inline articles/service-bus-messaging-overview.md]: sentence case

File: articles/service-bus-messaging-overview.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+The message sender can also require a variety of delivery characteristics including transactions, duplicate detection, time based expiration, and batching. These largely have postal analogies as well like repeat delivery, required signature, address change or recall.
+
+Azure Service Bus messaging has two separate features: Queues and Topics. Both support all of the concepts presented above - and more - the primary difference is that Topics support Publish-Subscribe capabilities that be used for sophisticated content based routing and delivery logic including sending to multiple recipients.
+
+For more about Service Bus Messaging
```
Comment (mollybostic) [inline articles/service-bus-messaging-overview.md]: make this Next Steps and sentence case

File: articles/service-bus-messaging-overview.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	ms.topic="article"
+	ms.date="02/20/2015"
+	ms.author="sethm"/>
+
+
+# Service Bus Messaging: Flexible Data Delivery in the Cloud
+
+Azure Service Bus messaging is a reliable information delivery service. The purpose of this service is to make communication easier. When two or more parties want to exchange information they need a communication mechanism. Azure messaging is a brokered, or third party communication mechanism. This is similar to a postal service in the physical world. Postal services make it very easy to send a variety of letters and packages with a variety of delivery guarantees anywhere in the world.
+
+Like the postal service delivering letters Azure Service Bus messaging is about flexible information delivery from both the sender and the recipient. The messaging service will ensure that the information is delivered even if the two parties are never both online at the same time, or if they aren't available at the exact same instance. In this way messaging is like sending a letter while non-brokered communication is like placing a phone call (or how a phone call used to be - before call waiting and caller ID, which are much more like brokered messaging).
```
Comment (mollybostic) [inline articles/service-bus-messaging-overview.md]: "Like the postal service delivering letters, Azure Service Bus..."

**Comment by mollybostic** [comment]
@sethmanheim to pull down and make updates

File: articles/service-bus-messaging-overview.md (hunk 2 above)
Comment (americanlimousine) [inline articles/service-bus-messaging-overview.md]: Whether you are planning a wedding or a night out on the town, renting Party Bus Service in DC for the kids on prom night, planning a wine tasting party, a bachelor or bachelorette party, if you need a Party Bus Service in DC, we here at American eagle limousine service will help ensure that your experience will be a memorable one that will be cherished for years to come. Get more details: http://americaneaglelimo.com/

**Pull request merged**